serverless/installing-openshift-serverless.adoc

```diff
@@ -15,31 +15,16 @@ toc::[]
 include::modules/serverless-cluster-sizing-requirements.adoc[leveloffset=+1]
 
 .Additional resources
-For more information on using the MachineSet API, read about xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc[Creating MachineSets].
-
+For more information on using the MachineSet API, read about xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#creating-machineset-aws[Creating MachineSets].
 
 // Add or remove an instance of a machine in a MachineSet
 include::modules/machineset-manually-scaling.adoc[leveloffset=+2]
 
-
 // Installing Service Mesh
 [id="installing-service-mesh_{context}"]
 == Installing Service Mesh
 
-An installed version of Service Mesh is required for the installation of {ServerlessProductName}. For details, see the {product-title} documentation on xref:../service_mesh/service_mesh_install/installing-ossm.adoc[Installing Service Mesh].
-
-[NOTE]
-====
-Use the Service Mesh documentation for Operator installation only. Once you install the Operators, use the documentation below to install the Service Mesh Control Plane and Member Roll.
```

Comment:
I believe we still need to keep the sentence in the docs: "Use the Service Mesh documentation for Operator installation only". The other sentence no longer applies. This is because for auto-installing Service Mesh Control Plane by the Serverless operator we need to have the Service Mesh operator already installed (on 4.1 this will not be done automatically).

Reply:
Thanks, Martin! :slightly_smiling_face: